src/driver/src/DefaultDriver.cpp

```diff
@@ -37,10 +37,10 @@
 
 #include <driver/DefaultDriver.hpp>
 
+namespace genny::driver {
 namespace {
 
 using namespace genny;
```

Comment:
Seems like this line can also be removed because all the usages of types are within the `genny` and `genny::driver` namespaces.

Reply:
yep